HTTP/1.1 200 OK
Server: nginx
Date: Sun, 24 Jul 2022 19:21:26 GMT
Content-Type: text/plain
Content-Length: 753
Last-Modified: Sun, 03 Feb 2019 21:19:52 GMT
Connection: keep-alive
ETag: "5c575af8-2f1"
Expires: Sun, 24 Jul 2022 23:00:00 GMT
Cache-Control: max-age=13114
Strict-Transport-Security: max-age=31536000; includeSubDomains
Accept-Ranges: bytes

From e8f3c86d93613eef5af187dfd2dba9ae9919a2ee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rapha=C3=ABl=20Proust?= <code@bnwr.net>
Date: Sun, 27 Jan 2019 13:31:28 +0800
Subject: [PATCH] Show bold not as bright

---
 x.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/x.c b/x.c
index 0422421..904e1dc 100644
--- a/x.c
+++ b/x.c
@@ -1296,10 +1296,6 @@ xdrawglyphfontspecs(const XftGlyphFontSpec *specs, Glyph base, int len, int x, i
 		bg = &dc.col[base.bg];
 	}
 
-	/* Change basic system colors [0-7] to bright system colors [8-15] */
-	if ((base.mode & ATTR_BOLD_FAINT) == ATTR_BOLD && BETWEEN(base.fg, 0, 7))
-		fg = &dc.col[base.fg + 8];
-
 	if (IS_SET(MODE_REVERSE)) {
 		if (fg == &dc.col[defaultfg]) {
 			fg = &dc.col[defaultbg];
-- 
2.20.1

